### PR TITLE
Make Fastlane for uploading to App Store connect work again #2306

### LINKED ISSUE #2306: Make Fastlane for uploading to App Store connect work again
Referring to https://docs.fastlane.tools/actions/upload_to_app_store/ (didn't check if that's the latest doc)

We are quite behind on `Fastlane` integration. Before we forked the code base, it was already set up, but I'm not sure if it ever worked. Let's try to get it to work for us.

PS: I noticed `submit_for_review` in `Deliverfile` is `true`. It (or it's replacement setting) should be set to `false`

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #2306

### CODE REVIEW

File: AlphaWalletTests/Foundation/DecimalNumberFormatterTest.swift [hunk 1]
```diff
@@ -38,10 +38,8 @@ class DecimalFormatterTest: XCTestCase {
         XCTAssertEqual(frDecimalFormatter.number(from: ",25"), NSNumber(value: 0.25))
     }
     func testNumberToStingInFr() {
-        //NOTE: For FR seems like it uses \u{202F} scalar for space instead of \u{00A0} like others do.
```
Comment (hboon) [inline AlphaWalletTests/Foundation/DecimalNumberFormatterTest.swift:41]: Can you add this fix in a separate PR?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -48,7 +48,7 @@ platform :ios do
     run_tests(
       workspace: "AlphaWallet.xcworkspace",
       scheme: "AlphaWalletTests",
-      device: "iPhone X",
+      device: "iPhone Xʀ",
```
Comment (hboon) [inline fastlane/Fastfile:51]: Why does switching to a different device help?

**Review by hboon — CHANGES_REQUESTED** [review]

File: fastlane/Fastfile (hunk 2 above)
Comment (vladyslav-iosdev) [reply fastlane/Fastfile]: yep, it seems like i didn't understand task.

**Comment by vladyslav-iosdev** [comment]
@hboon can u explain what exactly doesn't work here, and how can i reproduce it?

**Comment by hboon** [comment]
@vladyslav-iosdev sorry, was vague. Have updated the 1st comment in the issue #2306

**Pull request closed without merging**